recordlinkage/algorithms/nb_sklearn.py

```diff
@@ -211,8 +211,12 @@ def _fit_data(self, X):
             # initialise binarizer and save
             binarizer = LabelBinarizer()
 
+            if self.binarize:
+                binarizer.classes_ = numpy.array([0, 1])
```

Comment:
Numpy is imported as np.

Reply:
This is a very good catch. I fixed it and added a test.